src/erlfmt_cli.erl

```diff
@@ -196,51 +225,103 @@ read_stdin(Data) ->
+parse_opts([write | Rest], Files, Config) ->
+    parse_opts(Rest, Files, Config#config{out = replace});
+parse_opts([{out, _Path} | _Rest], _Files, #config{out = Out}) when Out =/= undefined ->
     {error, "out or replace mode can't be combined check mode"};
-parse_opts([{out, Path} | Rest], Name, Files, Config) ->
-    parse_opts(Rest, Name, Files, Config#config{out = {path, Path}});
-parse_opts([verbose | Rest], Name, Files, Config) ->
-    parse_opts(Rest, Name, Files, Config#config{verbose = true});
-parse_opts([check | _Rest], _Name, _Files, #config{out = Out}) when Out =/= standard_out ->
+parse_opts([{out, Path} | Rest], Files, Config) ->
+    parse_opts(Rest, Files, Config#config{out = {path, Path}});
+parse_opts([verbose | Rest], Files, Config) ->
+    parse_opts(Rest, Files, Config#config{verbose = true});
+parse_opts([check | _Rest], _Files, #config{out = Out}) when Out =/= undefined ->
     {error, "--check mode can't be combined write or replace mode"};
-parse_opts([check | Rest], Name, Files, Config) ->
-    parse_opts(Rest, Name, Files, Config#config{out = check});
-parse_opts([{print_width, Value} | Rest], Name, Files, Config) ->
-    parse_opts(Rest, Name, Files, Config#config{width = Value});
-parse_opts([require_pragma | _Rest], _Name, _Files, #config{pragma = insert}) ->
+parse_opts([check | Rest], Files, Config) ->
+    parse_opts(Rest, Files, Config#config{out = check});
+parse_opts([{print_width, Value} | Rest], Files, Config) ->
+    parse_opts(Rest, Files, Config#config{width = Value});
+parse_opts([require_pragma | _Rest], _Files, #config{pragma = insert}) ->
     {error, "Cannot use both --insert-pragma and --require-pragma options together."};
-parse_opts([require_pragma | Rest], Name, Files, Config) ->
-    parse_opts(Rest, Name, Files, Config#config{pragma = require});
-parse_opts([insert_pragma | _Rest], _Name, _Files, #config{pragma = require}) ->
+parse_opts([require_pragma | Rest], Files, Config) ->
+    parse_opts(Rest, Files, Config#config{pragma = require});
+parse_opts([insert_pragma | _Rest], _Files, #config{pragma = require}) ->
     {error, "Cannot use both --insert-pragma and --require-pragma options together."};
-parse_opts([insert_pragma | Rest], Name, Files, Config) ->
-    parse_opts(Rest, Name, Files, Config#config{pragma = insert});
-parse_opts([{files, NewFiles} | Rest], Name, Files0, Config) ->
-    parse_opts(Rest, Name, expand_files(NewFiles, Files0), Config);
-parse_opts([], _Name, [stdin], #config{out = Out}) when Out =/= standard_out, Out =/= check ->
+parse_opts([insert_pragma | Rest], Files, Config) ->
+    parse_opts(Rest, Files, Config#config{pragma = insert});
+parse_opts([{files, NewFiles} | Rest], Files0, Config) ->
+    parse_opts(Rest, expand_files(NewFiles, Files0), Config);
+parse_opts([], [stdin], #config{out = Out}) when
+    Out =/= standard_out, Out =/= undefined, Out =/= check
```

Comment:
Can `Out` ever be `standard_out` now?

Reply:
I do not know what I was thinking.
I got confused with `-o`
I will remove `undefined`
Sorry about that.